File: github-enterprise-server/TBL-GithubEnterpriseServerEvidenceCollector-DotNet/SampleApp_GithubEnterpriseServer/Models/GitHubPullRequestCommentsResponseModel.cs
using System;

namespace SampleApp_GithubEnterpriseServer.Models
{
    /// <summary>
    /// Response model used for deserializing the JSON response from
    /// the GitHub Enterprise Server API
    /// </summary>
    public class GitHubPullRequestCommentsResponseModel
    {
        public User user { get; set; }
        public DateTime created_at { get; set; }
        public string author_association { get; set; }
        public string body { get; set; }
    }
}

File: github-enterprise-server/TBL-GithubEnterpriseServerEvidenceCollector-Java/src/com/tugboatlogic/sample/models/GitHubPullRequestCommentsResponseModel.java
package com.tugboatlogic.sample.models;

/*
 * Response model used for deserializing the JSON response from
 * the GitHub Enterprise Server API
 */
public class GitHubPullRequestCommentsResponseModel {
	public String created_at;
	public String author_association;
	public String body;
	public User user;
}
